packages/Modal/src/components/Header/Header.styles.js

```diff
@@ -6,8 +6,8 @@ import Heading from "@paprika/heading";
 export const Header = styled.div`
   align-items: center;
   display: flex;
+  height: ${spacer(7)};
   justify-content: space-between;
-  min-height: ${spacer(7)};
   padding: 0 ${spacer(2)};
```

Comment:
I think we should use `min-height: ${spacer(7)};` but adjust the padding size, to make the calculated height equals to 56px, what do you think